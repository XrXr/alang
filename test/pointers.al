main :: proc() {
    var byte u8
    byte = 253
    a := 3
    ap := &a
    bytep := &byte
    @ap = 80
    @ap = @ap + 80
    @bytep = 123
    a = a - 3
    a = a - byte
    puts("should be 34\n")
    print_int(a)
}
