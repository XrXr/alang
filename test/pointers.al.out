should be 34
34
